app/assets/javascripts/answers.js

```diff
@@ -20,12 +20,23 @@ $(document).ready(function(e){
   $("#answers").hide();
   $("#response").hide();
 
+   if($('#answer_text').val() ==  "") 
+   $('#submit_answer').attr('disabled', true);
+
+  $('#answer_text').keyup(function(){
+    if($('#answer_text').val() !=  "") 
+         $('#submit_answer').attr('disabled', false);    
+    else
+         $('#submit_answer').attr('disabled', true);   
```

Comment:
A more concise way to write this 4-line block of code in a single line would be, e.g.:
```
$('#submit_answer').attr('disabled', !($('#answer_text').val()));
```